Makefile

```diff
@@ -255,21 +253,50 @@ coverage-report-html:
 	go tool cover --html=./cover.out
 
 build: fmt lint ## Build manager binary.
-	go build -ldflags "-X 'main.buildVersion=${BUILD_VERSION}' -X 'main.buildDate=${BUILD_DATE}'" -mod vendor -o bin/manager main.go
+	GOARCH=${GOARCH} go build -ldflags "-X 'main.buildVersion=${BUILD_VERSION}' -X 'main.buildDate=${BUILD_DATE}'" -mod vendor -o bin/manager main.go
 
 image-build: ## Build operator OCI image.
 	$(OCI_BIN) build --build-arg BUILD_VERSION="${BUILD_VERSION}" -t ${IMG} .
 
-ci-images-build: image-build
-	$(OCI_BIN) build --build-arg BASE_IMAGE=$(IMG) -t $(IMG_SHA) -f ./shortlived.Dockerfile .
-
 image-push: ## Push OCI image with the manager.
 ifneq (,$(findstring quay.io/netobserv/,$(IMG)))
 	$(error Do not push to quay.io/netobserv)
 else
 	$(OCI_BIN) push ${IMG}
 endif
 
+single-multiarch-linux-build/%:
+#The --load option is ignored by podman but required for docker
+	DOCKER_BUILDKIT=1 $(OCI_BIN) buildx build --load --build-arg TARGETPLATFORM=linux/$* --build-arg TARGETARCH=$* --build-arg BUILDPLATFORM=linux/amd64 -t ${IMG}-$* -f Dockerfile .
+
+single-multiarch-linux-push/%: single-multiarch-linux-build/%
+	DOCKER_BUILDKIT=1 $(OCI_BIN) push ${IMG}-$*
```

Comment:
I think we should keep build and push separate and if needed we can have target to do both